### DISCUSSION TITLE
布谷TV 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#130] [模板讨论]
布谷TV 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by hecarli555** [comment]
请问提示如下内容是什么意思？
<html>
<body>


Failed at 1/5 request,Fail assert: {"re": "200", "from": "status"} from success_asserts,Response Error : HTTP 301: Moved Permanently,Request URL: https://www.bugutv.net/wp-admin/admin-ajax.php
--



</body>
</html>

**Reply by wjf0214** [reply]
网站域名换了，请更新最新的模板。

**Reply by icedog111** [reply]
现在是这个错误，应该怎么调整
Failed at 1/5 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: error:0A000126:SSL routines::unexpected eof while reading,
Request URL: https://www.bugutv.org/wp-admin/admin-ajax.php

**Reply by wjf0214** [reply]
网络请求失败了。

**Reply by xuexijiaoben** [reply]
这两天一直假签到，手动执行可以看到签到成功，自动就假签到

**Reply by hecarli555** [reply]
没发现布谷域名改了么？？？删除模板，更新公共模板，然后再重新订阅布谷的签到吧

**Comment by hecarli555** [comment]
求修复
Failed at 1/5 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://www.bugutv.org/wp-admin/admin-ajax.php

**Comment by hecarli555** [comment]
今天又开始报错了，求大佬修复，万分感谢
Failed at 1/5 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://www.bugutv.vip/wp-admin/admin-ajax.php

**Comment by hecarli555** [comment]
布谷开了cf盾。。。。怎么破？
